specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json

```diff
@@ -122,7 +122,12 @@
   "definitions": {
     "CloudError": {
       "x-ms-external": true,
-      "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse",
+      "properties": {
```

Comment:
The original is right, why did you put the $ref into an "error" object

Reply:
Because of this comment: https://github.com/Azure/azure-rest-api-specs/pull/10986#issuecomment-731369102

I was told it had to be like this [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2019-09-01/policyAssignments.json#L585](url)

@colawwj  could you please clarify which version is the correct one?

Reply:
it is **error** the property that should refer to "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse" not the CloudError itself.

Reply:
@feruilob , please change the `$ref` to `../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse` as [this file](https://github.com/Azure/azure-rest-api-specs/blob/bf8027fc7b7a4f4fe4cb1388678900a59224df76/specification/resources/resource-manager/Microsoft.Authorization/stable/2019-09-01/policyAssignments.json#L585). 
@colawwj , can you please investigate what's the purpose of `x-ms-external` extension and what's the impact of removing it? Thanks.

Reply:
@raych1 done

Reply:
@jianyexi , can you help with lintDiff error?

Reply:
@raych1 
please change the  ```$ref``` to ```../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse``` in https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json#L72  and  https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json#L112 , and delete property of ```CloudError```  in  https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json#L123

Reply:
@colawwj , do you confirm if your suggestion is the only approach found so far for the fix?

Reply:
@raych1 
We have test the approach like ```$ref``` to``` ../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse```,but it still reports an error the same as before.
Cc : @qiaozha

Reply:
@feruilob , could you take @colawwj's advice for the fix?

Reply:
Alright. Doing this for other files in  Microsoft.Authorization/stable/2020-09-01 and 2020-03-01 also

Reply:
Just confirming: it's v2 not v1, right? ../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse